styles/japanesewithanime.com/catppuccin.user.css

```diff
@@ -0,0 +1,397 @@
+/* ==UserStyle==
+@name japanesewithanime.com Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/japanesewithanime.com
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/japanesewithanime.com
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/japanesewithanime.com/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Ajapanesewithanime.com
+@description Soothing pastel theme for japanesewithanime.com
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red*", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+@var select linkColor "Link color" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue*", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+@var checkbox solidColors "Prefer solid colors with no borders" 0
```

Comment:
Pick one ideally, maybe the one that most matches the original site's appearance.